circle.yml

```diff
@@ -99,7 +99,8 @@ jobs:
           set -e
           export GO111MODULE=off
           export GOPATH=/tmp/gopath
-          mkdir -p $GOPATH/src
+          mkdir -p $GOPATH/src/github.com/gopherjs/gopherjs
+          cp -r -p . $GOPATH/src/github.com/gopherjs/gopherjs/
```

Comment:
Could you explain why this is necessary?

Reply:
TodoMVC test
- use GOPATH mode. github.com/gopherjs/todomvc go get  use github.com/gopherjs/gopherjs main version, not current PR version
- use GOMOD mod. github.com/gopherjs/todomvc use github.com/gopherjs/gopherjs current PR mode. 
- GOPATH mode and GOMOD mode diffrent github.com/gopherjs/gopherjs/nosync test failed.

use GOPATH mode, copy PR version to GOPATH and build.

Reply:
Ah, good catch, thank you for the fix.